pom.xml

```diff
@@ -70,7 +70,7 @@
               <overwrite>true</overwrite>
               <resources>
                 <resource>
-                  <directory>${project.basedir}/configs/openmrs_config</directory>
+                  <directory>${project.basedir}/configs/openmrs</directory>
```

Comment:
There can be additional types of configurations for OpenMRS, so it would be reasonable to allow for this, such as:
`directory>${project.basedir}/configs/openmrs/initializer</directory>`